### DISCUSSION TITLE
当我使用8bit并口显示屏时,我无法使用触摸

### DISCUSSION THREAD

**Discussion by zhanshenquan** [Bodmer/TFT_eSPI#1045] [Q&A - Fonts] [answered]
在没有使用触摸功能的情况下,可以正常下载程序,当我反注释[#define TOUCH_CS 23     // Chip select pin (T_CS) of touch screen]和[// The ESP32 has 2 free SPI ports i.e. VSPI and HSPI, the VSPI is the default.
// If the VSPI port is in use and pins are not accessible (e.g. TTGO T-Beam)
// then uncomment the following line:
#define USE_HSPI_PORT]后编译器[Arduino]会报错
请帮助我使用8bit并口显示屏时,同时使用触摸,我的触摸驱动芯片是xpt2016
![图片一](https://user-images.githubusercontent.com/61562302/109959869-08ce5780-7d23-11eb-88f8-fc22b40940dc.PNG)

**Comment by Bodmer** [comment, marked as answer]
I am not sure what the question is. Can you ask in English?

Post a sketch that deomstrates the problem, describe your hardware setup and post the complete setup file you are using.
